packages/blade/src/components/AutoComplete/_decisions/decisions.md

```diff
@@ -0,0 +1,240 @@
+        <ActionListItem title="Hyderabad" value="hyderabad" />
+      </ActionList>
+    </BottomSheetBody>
+  </BottomSheet>
+</Dropdown>
+```
+
+### With Tags Outside
+
+When possible selections are more than 11, it is advised to put tags outside of the Input.
+
+<img width="700" alt="image" src="https://github.com/razorpay/blade/assets/30949385/8906eb6c-fae0-4e99-bd3f-d7e0edc60537">
+
+We add `showTags={false}` to AutoComplete to remove tags from input. Then we can use Controlled state to show tags outside in the UI.
+
+If not tags, what to show inside AutoComplete :thinking:? Checkout this table to understand -
+
+| Code                                                            | Output                                                                                                                     |
+| --------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------- |
+| `<AutoComplete label="Status" showTags={false} />`              | <img width="220" alt="image" src="https://github.com/razorpay/blade/assets/30949385/c8a30845-812c-493f-a51f-97c233880960"> |
+| `<AutoComplete accessibilityLabel="Status" showTags={false} />` | <img width="200" alt="image" src="https://github.com/razorpay/blade/assets/30949385/83cf63d5-ccc4-4bb9-bf0d-ce91dc5c1b09"> |
+
+So when input has label, we show number of selected items.
+
+When input doesn't have a label, we show accessibilityLabel inside the input.
+
+**Code**
+
+```jsx
+const [selectedItems, setSelectedItems] = React.useState([]);
+
+// ...
+
+<AutoComplete
+  accessibilityLabel="Status"
+  rows="1"
+  showTags={false} // we don't want to show tags at 2 places
+  selected={selectedItems}
+  onSelectChange={({ values }) => setSelectedItems(values)}
+/>
+
+
+// Somewhere outside in your UI
+<Box>
+  {selectedItems.map((selectedItemName, index) =>
+    <Tag
+      onDismiss={() => removeItem(index)}
+    >
+      {selectedItemName}
+    </Tag>
+  )}
+</Box>;
+```
+
+### With Different Inactive - Active States
+
+| Code                                 | Inactive                                                                                                                   | Active                                                                                                                     |
+| ------------------------------------ | -------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------- |
+| `<AutoComplete rows="1" />`          | <img width="266" alt="image" src="https://github.com/razorpay/blade/assets/30949385/74b849f3-f1b9-4922-b717-70ffc7ffd56e"> | <img width="276" alt="image" src="https://github.com/razorpay/blade/assets/30949385/055bc4f4-fb57-4129-b27a-0f1966c8ad26"> |
```

Comment:
add some description that the number of tags shown will be dependent on the width of the Input?